include/igl/copyleft/comiso/miq.cpp

```diff
@@ -612,10 +616,8 @@ IGL_INLINE void igl::copyleft::comiso::PoissonSolver<DerivedV, DerivedF>::solveP
 
   clearUserConstraint();
   // copy the user constraints number
-  for (size_t i = 0; i < hardFeatures.size(); ++i)
-  {
-    addSharpEdgeConstraint(hardFeatures[i][0],hardFeatures[i][1]);
-  }
+  for (auto element : hardFeatures)
```

Comment:
Can you keep the braces around the `for` loop, assert that `element` has size 2 and change the `auto` into `const auto &` (since it's another `std::vector` inside).

Reply:
done, thanks I have overlooked this one